nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/main/java/org/nuxeo/elasticsearch/audit/ESAuditMigrationWork.java

```diff
@@ -77,12 +79,12 @@ public void work() {
         TransactionHelper.commitOrRollbackTransaction();
             long t0 = System.currentTimeMillis();
             long nbEntriesMigrated = 0;
-            int pageIdx = 1;
+            pageIdx = 1;
```

Comment:
Please don't make it a field. The issue is that the lambda needs final or effectively final captured variables, which is not the case of this one. The solution is, just before the lambda, to create `int pageIdxF = pageIdx` and use `pageIdxF` inside the lambda.